java/org/apache/catalina/tribes/group/GroupChannel.java

```diff
@@ -335,7 +335,7 @@ public void messageReceived(ChannelMessage msg) {
                 if (channelListener != null && channelListener.accept(fwd, source)) {
                     channelListener.messageReceived(fwd, source);
                     delivered = true;
-                    //if the message was accepted by an RPC channel, that channel
+                    //if the message was accepted by a RPC channel, that channel
                     //is responsible for returning the reply, otherwise we send an absence reply
```

Comment:
This is correct. It is "an RPR channel" because R has a [vowel sound ](https://www.quora.com/Which-is-correct-in-English-an-R-or-a-R) when pronounced.